mixlib-shellout-x86-mingw32.gemspec

```diff
@@ -1,9 +0,0 @@
-# x86-mingw32 Gemspec #
```

Comment:
Blargh...  I tried to get bundler to conditionally recognize/install platforms but that seems to not be currently supported.  Either that or I seem to not understand all the interdependencies between rake and gem and bundler.